app/react/Thesauri/Notice.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import { Icon } from 'app/UI';
+
+function learnIcon() {
+  return (
+    <svg width="29" height="29" viewBox="0 0 29 29" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Does this need to be added to font awesome as a custom icon? Font awesome needs a single svg path, so we would have to regenerate this.